README.md

```diff
@@ -125,25 +125,31 @@ https://github.com/reactstrap/code-sandbox-examples
 Install dependencies:
 
 ```sh
-npm install
+yarn install
 ```
 
 Run examples at [http://localhost:8080/](http://localhost:8080/) with webpack dev server:
 
 ```sh
-npm start
+yarn start
+```
+
+Run tests:
+
+```sh
+yarn test
 ```
 
-Run tests & coverage report:
+Get test coverage report:
 
 ```sh
-npm test
+yarn cover
 ```
 
 Watch tests:
 
 ```sh
-npm run test-watch
+yarn test --watchAll
```

Comment:
I can remove that. `yarn test` requests further user input for me after running, so I added the watchAll flag, since the "Watch tests" section was there originally, but... I kind of thought it was redundant.